.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
```

Comment:
Will this automatically change the title with what is in title? If yes, that's really cool!

Reply:
Yes it will. Quite nice change. Also, the label is automatically applied.